### ISSUE TITLE
Invalid path alias error with Craft 3.6

### ISSUE THREAD

**Issue by RyanRoberts** [FosterCommerce/snipcart-craft-plugin#29]
I have just installed a fresh copy of Craft 3.6 and Snipcart 1.5.0 and it looks like something's not happy. 

Here's what I'm getting:

```
Invalid Argument – yii\base\InvalidArgumentException

Invalid path alias: @workingconcept/snipcart/assetbundles/dist/img/disconnected.svg
```

Here's a screenshot https://cln.sh/LJPXzg

🤔

**Comment by sjcallender** [comment]
Just released a fix for this in https://github.com/FosterCommerce/snipcart-craft-plugin/releases/tag/1.5.1

**Comment by tobystokes** [comment]
composer still thinks latest is 1.5.0, just checking you're aware of this?

**Comment by sjcallender** [comment]
@tobystokes, just released 1.5.1.1 to fix the composer version issue.
